deployments.go

```diff
@@ -37,7 +37,7 @@ type DeploymentsCreateOpts struct {
 // deployer is an interface that represents something that can perform a
 // deployment.
 type deployer interface {
-	Deploy(context.Context, DeploymentsCreateOpts) (*Release, error)
+	Deploy(context.Context, *DeploymentsCreateOpts) (*Release, error)
```

Comment:
Yea, this is just me being not very familiar with go. I wanted to return nil here: https://github.com/remind101/empire/pull/622/files#diff-71bb861dacfb9ab12070c76ed3a8ce95R37, but it makes sense that the Deploy func should be required to take the struct. Let me fix that.